combat-trainer.lic

```diff
@@ -2223,6 +2233,33 @@ class GameState
     end
   end
 
+  def tactics_analyze?
+    return false if DRSkill.getxp('Tactics') > 32
```

Comment:
Reading the code, it looks like this method will always analyze, even if `@tactics_combo_array` is already populated. Is that the case?